src/dashboard/Data/Browser/DataBrowser.react.js

```diff
@@ -591,7 +591,7 @@ export default class DataBrowser extends React.Component {
             <ResizableBox
               width={this.state.panelWidth}
               height={Infinity}
-              minConstraints={[100, Infinity]}
+              minConstraints={[400, Infinity]}
```

Comment:
Please revert if not required for this PR; the panel should be resizable to min. 100 px width.